src/components/Bank.js

```diff
@@ -33,10 +39,17 @@ function hasExchange(sell, buy) {
   return true;
 }
 
+function currencyFormat(price) {
+  const parsePrice = Number.parseFloat(price).toFixed(2);
```

Comment:
great work, but we could use the lang to do it for us.

```js
const number = 12345.67;
number.toLocaleString('es-AR', {currency: 'ARS', style: 'currency'});
// result "$ 12.345,67"
```

If you wonder how `toLocaleString` works check it here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString.
Under the hood it's calling https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat

Reply:
Thanks, will do